src/apps/material/material.tsx

```diff
@@ -45,6 +45,10 @@ const Material: React.FC<MaterialProps> = ({ wid }) => {
   const [currentManifestation, setCurrentManifestation] =
     useState<ManifestationsSimpleFieldsFragment | null>(null);
 
+  // periodikumSelect must be used later to change the UI and reservation when you have chosen a specific periodical
+  // eslint-disable-next-line
+  const [periodikumSelect, setPriodikumSelect] = useState<string | null>(null);
```

Comment:
Please update this code to use periodical instead of periodikum (including typos).

:+1: for adding the explanation for the line. [Please refer to the specific rule being disabled instead of disabling eslint entirely for the line](https://eslint.org/docs/latest/user-guide/configuring/rules#using-configuration-comments-1).